config/routes.rb

```diff
@@ -3,9 +3,11 @@
   mount DatastoreApi::HealthEngine::Engine => '/datastore'
 
   unless HostEnv.production?
-    require 'sidekiq/web'
-    require 'sidekiq-scheduler/web'
-    mount Sidekiq::Web => "/sidekiq"
+    authenticate :user, lambda { |u| u.admin? } do
```

Comment:
User manager and admin have the same definition, why not use user manager? or do we plan to differentiate in the future?

Reply:
I think admin makes more semantic sense here, than user_manager, and it might be useful to differentiate if we wanted to change it. Especially because in this context we would only want the devs to access Sidekiq, I think the intention is clearer if it's admin.